### ISSUE TITLE
Fixed scaling not working in ARQuickLook when auto generating USDZ

### ISSUE THREAD

**Issue by milesgreen** [google/model-viewer#2405]
### Description

Auto-generated USDZ is amazingly awesome. Thanks for all the work that has gone into that @elalish, @kolodi, @mrdoob !

While testing out this feature, it appears that setting `ar-scale="fixed"` in conjunction with using auto-generated USDZ, is no longer forcing the scaling to be fixed in QuickLook. The user is able to scale the object. 

When not using auto generation and providing an `ios-src` USDZ path and setting `ar-scale="fixed"`, this still works, and the user is unable to scale object in QuickLook. 

#### Live Demo

https://muddy-pinnate-rutabaga.glitch.me/

### Version

- model-viewer: v1.7.1

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [x] Safari

### OS

- [ ] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [x] QuickLook

**Comment by elalish** [comment]
Yes, I believe this is because Safari does not handle query params on Blob URLs. I believe this means the whole Quick Look API is killed off, which is unfortunate. Please file a bug on Apple.

**Comment by maciekglowka** [comment]
Thanks for pointing this issue.
I've had recently a problem with auto-generated usdz not being displayed in quick look at all. The spinner just keeps spinnig.
It seems that removing ar-scale param solves the problem for now.
Although I'd like to have a fixed scale eventually.
(I've seen the problem on iOS 15.2)

http://tests.glowka.atthost24.pl/ - a small test with ar-scale set to fix, where the problem can be seen.

**Comment by chaitanya-deep** [comment]
Happened to me too.

Took me a while to realize `ar-scale` is crashing the quick look on iOs. Would it be helpful to add this as a note in the documentation?

**Comment by maciekglowka** [comment]
This, or do not attach it to usdz blob for now.

**Comment by elalish** [comment]
Indeed, I can repro. Actually I had a similar experience when I loaded https://modelviewer.dev/examples/scenegraph/#variants the first time in Quick Look (it took ages, but did eventually work), but the second time it was fast (that one does not have `ar-scale` set. But yes, it seems markedly worse with the `ar-scale` attribute. Agreed we should probably just disable this until Apple fixes it. Would any of you be so kind as to file a bug or two on https://bugs.webkit.org/ about this? That's the best place I've found to report Quick Look issues.

**Comment by maciekglowka** [comment]
Thanks
Reported: https://bugs.webkit.org/show_bug.cgi?id=236069

**Comment by elalish** [comment]
@grorg Here's a QuickLook issue for you, and it even has a webkit bug! For everyone else, @grorg works at Apple and said he'd love to be notified of any Apple-related problems we have (thank you!), so please feel free to @ mention him.

**Comment by milesgreen** [comment]
Fixed scaling when using auto USDZ now works in Safari iOS 15.5
